threescale/api/utilities.go

```diff
@@ -129,6 +130,24 @@ func (ur UsageReport) IsSame(usageReport UsageReport) bool {
 	return true
 }
 
+// OrderByAscending sorts each slice in the usage reports in order of ascending granularity
+func (urs UsageReports) OrderByAscending() {
```

Comment:
maybe use `OrderByAscendingGranularity`? This way we make it very clear we don't take timestamps into consideraton.